### ISSUE TITLE
Add ENA study accession + link to S-SUBS10

### ISSUE THREAD

**Issue by ESapenaVentura** [ebi-ait/hca-ebi-wrangler-central#648]
ENA accession: PRJEB38994
BioStudies accession: S-SUBS10

**Comment by ke4** [comment]
I can't see S-SUBS10 in BioStudies Submission Tool where I could edit it.
I can only see 1 project there: s_SUBS3 and even that I can't edit as the tool just gives me an error when I am trying to edit it.
I contacted with the BioStudies Team and waiting for their reply.

**Comment by ke4** [comment]
BioStudies Team suggested that as a one-off they can edit their DB to help us. I am not sure that is the best idea.

**Comment by ESapenaVentura** [comment]
I mean, what other option do we have? it's their service... I hoped we could do it but if there is no other way, we can get the contributor's request this way

**Comment by ke4** [comment]
I passed the ENA accession information to BioStudies Team.

**Comment by gabsie** [comment]
IN a chat with Biostudies team about ownership.

**Comment by ke4** [comment]
After a password reset I managed to login to the BioStudies Submission Tool.
It told me that this submission (S-SUBS10) is not complete. It is missing the `Organism` field.

I set the `Organism` to 'Homo sapiens (human)` and added the ENA project accession and re-submitted the study.
After submission I checked it, but the added values/links are not there yet. We got a message that processing could take up 24 hours.
